packages/console/src/pages/ApiResourceDetails/index.tsx

```diff
@@ -87,10 +87,12 @@ const ApiResourceDetails = () => {
         <>
           <Card className={styles.header}>
             <div className={styles.info}>
-              <ImagePlaceholder size={76} borderRadius={16} />
+              <div className={styles.imagePlaceholder}>
+                <ImagePlaceholder size={60} borderRadius={16} />
```

Comment:
Why doesn't this component use CSS class? I know that's not quite related to this PR, but I don't think the props and inline styles are OK

Reply:
Because the size of `img` inside the `ImagePlaceholder` will change according to the size of `ImagePlaceholder`.

Reply:
Even this we can still use CSS to set the size of the image. But I understand it's a temp solution at this point, will refactor it later

Reply:
resolved.
https://silverhand-io.slack.com/archives/C02CMB8JVJB/p1650502950837229